plugins/examples/pod-counter/src/index.tsx

```diff
@@ -1,24 +1,30 @@
-import React from 'react';
-import { Headlamp, Plugin, Registry, K8s } from '@kinvolk/headlamp-plugin/lib';
+import { Headlamp, K8s,Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
```

Comment:
npm run format yields changes here.

Reply:
I thought I might as well just push the updated commit since it's a simple change.